common/djangoapps/util/milestones_helpers.py

```diff
@@ -116,6 +118,22 @@ def get_prerequisite_courses_display(course_descriptor):  # pylint: disable=inva
     return pre_requisite_courses
 
 
+def collect_course_milestone(course_key, user):
```

Comment:
nitpick: is "collect" the best word for this function name. Would "fulfill" be better?